src/lib/Microsoft.Health.Extensions.Fhir/Search/SearchExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static string CompositeOr(this IEnumerable<string> searchTokens)
         /// </summary>
         /// <param name="searchTokens">Enumerable of search tokens to combine.</param>
         /// <returns>A concatenated string of search terms joined for an And search.</returns>
-        /// <seealso cref="https://www.hl7.org/fhir/search.html#combining"/>
+        /// <seealso cref="!:https://www.hl7.org/fhir/search.html#combining"/>

```

Comment:
What does the `!:` do? Just curious

Reply:
It was to syntactically represent urls in a cref (without that the compilation fails). But I realized that it doesn't actually hyperlink it, and only renders the text. So, I changed to href.